src/Converter.php

```diff
@@ -211,19 +211,24 @@ protected static function getClassesMapping(): array
 
 		if (!empty($classes)) {
 			$mandatory_keys = [
-				RB::KEY_HANDLER,
-				RB::KEY_KEY,
+				RB::KEY_HANDLER => [TYPE::STRING],
+				RB::KEY_KEY => [TYPE::STRING, TYPE::NULL],
 			];
 			foreach ($classes as $class_name => $class_config) {
 				if (!\is_array($class_config)) {
 					throw new Ex\InvalidConfigurationElementException(
 						sprintf("Config for '{$class_name}' class must be an array (%s found).", \gettype($class_config)));
 				}
-				foreach ($mandatory_keys as $key_name) {
+				foreach ($mandatory_keys as $key_name => $allowed_types) {
 					if (!\array_key_exists($key_name, $class_config)) {
 						throw new Ex\IncompleteConfigurationException(
 							"Missing '{$key_name}' entry in '{$class_name}' class mapping config.");
 					}
+
+                    if (!\in_array(\gettype($class_config[$key_name]), $allowed_types)) {
```

Comment:
We can have this whole check simplified by using `Validator` class: `Validator::assertIsType(RB::KEY_KEY, $class_config[$key_name], $allowed_types);` - the error message may not be as pretty but we get less code.

Reply:
@MarcinOrlowski I have just updated this!